src/Components/MyReservations.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchReservations, removeReservation } from '../Redux/reservations/myReservationsSlice';
+import '../MyReservations.css';
+
+function MyReservations() {
```

Comment:
- Please, consider using arrow funcs according to Javascript ES6 good practices.